tests/README.md

```diff
@@ -396,3 +396,28 @@ uvm.help.tmux_ssh()
 
 It supports a number of options, you can check with `devtool sandbox --
 --help`.
+
+### Running outside of Docker
+
+Running without Docker
+
+```
+source /etc/os-release
+case $ID-$VERSION_ID in
+al2)
```

Comment:
This seems off. "al2" does not contain a dash, but the thing we're cas-ing on seems to contain one?

Reply:
True. I probably fumbled it.

Reply:
Fixed